testsuite/MDAnalysisTests/analysis/test_rms.py

```diff
@@ -128,6 +128,44 @@ def test_with_superposition_equal(self):
         rmsd_superposition = rms.rmsd(A, B, center=True, superposition=True)
         assert_almost_equal(rmsd, rmsd_superposition)
 
+class TestRMSD(object):
+    def setUp(self):
+        self.universe = MDAnalysis.Universe(GRO, XTC)
```

Comment:
`Universe(PSF, DCD)` loads much faster than the GRO file.

To cut down on processing, use `start`, `stop`, and `step`, with the additional advantage that they get tested, too.